tests/templates/requestTestData/middleware.ts

```diff
@@ -93,7 +93,6 @@ export const testSet: TestSet = {
 				data: '',
 				headers: {
 					location: 'http://localhost/somewhere-else?redirect=',
-					'x-matched-path': '/api/hello',
```

Comment:
this is related to fixing not exiting routing to make a redirect when continue is true (first commit). fix was in part to ensure that redirects are entered properly for i18n, but also just because I'm pretty sure that's how I should have done it in the first place.

Reply:
awesome, thanks for clarifying this :slightly_smiling_face::+1: